packages/sign_in_with_apple/example/pubspec.lock

```diff
@@ -1,48 +1,62 @@
 # Generated by pub
 # See https://dart.dev/tools/pub/glossary#lockfile
 packages:
+  archive:
```

Comment:
@tp what flutter version do you have?

Reply:
Same, when I run `flutter packages get`:

```
  archive:
    dependency: transitive
    description:
      name: archive
      url: "https://pub.dartlang.org"
    source: hosted
    version: "2.0.11"
  args:
    dependency: transitive
    description:
      name: args
      url: "https://pub.dartlang.org"
    source: hosted
    version: "1.5.2"
```